compiler/src/scala/quoted/runtime/impl/printers/SourceCode.scala

```diff
@@ -30,6 +30,7 @@ object SourceCode {
     if (flags.is(Flags.Enum)) flagList += "enum"
     if (flags.is(Flags.Erased)) flagList += "erased"
     if (flags.is(Flags.ExtensionMethod)) flagList += "extension"
+    if (flags.is(Flags.Exported)) flagList += "exported"
```

Comment:
We need a similar addition in `compiler/src/scala/quoted/runtime/impl/printers/Extractors.scala`

Reply:
Fixed